Amend "need to re-join" guidance to align with practice (#391)
w3c/Guide: process/charter.md

  
  1. A w3.org URI to the proposed charter (not a github.io URI). This charter is public, and must not be altered, during the AC review.
  2. The list of significant changes to a revised charter (per ["Advisory Committee Review of a Working Group or Interest Group Charter"](https://www.w3.org/policies/process/#CharterReview) of the Process Document). It is useful to include a diff between the current and proposed charters (you may wish to use the [HTML diff tool](https://services.w3.org/htmldiff)).

>   3. In case of renewal of an existing charter, whether the current group participants would need to re-join the group once the revised charter is approved. Group participants would need to re-join if the scope changes.

Reviewer: ```suggestion
  3. When renewing an existing charter, whether the current group participants would need to re-join the group once the revised charter is approved. Group participants would need to re-join if the scope changes.
```
